[native]: Enable Velox cuDF (#25094)
prestodb/presto: presto-native-execution/README.md

`./scripts/setup-adapters.sh arrow_flight`

#### Nvidia cuDF GPU Support

To enable support with [cuDF](https://github.com/facebookincubator/velox/tree/main/velox/experimental/cudf),
add to the extra cmake flags:
`EXTRA_CMAKE_FLAGS = -DPRESTO_ENABLE_CUDF=ON`

> In some environments, the CUDA_ARCHITECTURES and CUDA_COMPILER location need to be explicitly set.

Reviewer: ```suggestion
In some environments, the CUDA_ARCHITECTURES and CUDA_COMPILER location must be explicitly set.
```

The make command will look like:

> `CUDA_ARCHITECTURES=80 CUDA_COMPILER=/usr/local/cuda/bin/nvcc EXTRA_CMAKE_FLAGS=" -DPRESTO_ENABLE_CUDF=ON" make`

Reviewer 2: Just FYI. The VELOX CI uses `CUDA_ARCHITECTURES=70` because this is the minimum support architecture.

Reviewer 3: we can change that to 70, but shouldn't we actually change it to say "native", so it gets the best possible support for the architecture that is present at compilation time? or do we always want to default the lowest possible architecture?

Author: This is only an example usage in README. GPU users would build with what is best for them.
